dali/base/dadfs.cpp

```diff
@@ -2782,6 +2790,14 @@ class CDistributedFileBase : implements INTERFACE, public CInterface
         root->removeProp("Attr");
         return NULL;
     }
+    IPropertyTree * resetHistory(IPropertyTree *history=NULL)
```

Comment:
What purpose does the parameter serve? Not used below other than to set to blank vs delete.